### ISSUE TITLE
Feature Request: Allow for insertion of templates

### ISSUE THREAD

**Issue by Tejeev** [aptend/typing-transformer-obsidian#77]
I'm looking for a quick way to add templates in line.  Currently I copy the template into the Typing Transformer config, but it would be cool if I could just have it just insert from template so each could be edited atomically.

**Comment by aptend** [comment]
Could you give me some specific examples? I'm unclear about what 'templates' means; is it the core plugin Templates?

**Comment by Tejeev** [comment]
I think it's prudent to store the templates in the default configured template directory but basically I could add some lines of text to a file saved in the template dir then in the Typing Transformer config, specify as string which would then be replaced with the text from that file
So for example, instead of:
```
'tgg..|' -> '---\n### ⭐: `INPUT[toggle:starred]`\n#### Open `INPUT[toggle:closed]` Closed\n#### Currently Working `INPUT[toggle:tabled]` Tabled: `INPUT[inlineSelect(option(Engineering), option(Customer), option( Me ), option(Closed), option(Someone Elses)):status]` \n#### Summary: `INPUT[textArea:summary]`\n---|'
```
I could instead have a file `toggles` in the default templates dir which contains the following:
```
---
### ⭐: `INPUT[toggle:starred]`
#### Open `INPUT[toggle:closed]` Closed
#### Currently Working `INPUT[toggle:tabled]` Tabled: `INPUT[inlineSelect(option(Engineering), option(Customer), option( Me ), option(Closed), option(Someone Elses)):status]`
#### Summary: `INPUT[textArea:summary]`
---
```
Then in the TT config, just the following line:
`'tgg..|' -> toggles'|'`

**Comment by aptend** [comment]
@Tejeev 0.4.2 has been released; you might want to give it a try

**Comment by Tejeev** [comment]
Hi @zanodor,  thank you, but I do not use VSCode.  I don't think I've edited anything plain text outside of vim except for in Obsidian for like 5 years, lol!  It makes Go marginally easier, but I've got most of the functionality in vim+plugins and a compiler. 
Anyway, the benefit to templates is that it makes it easier to follow the philosophy of each object doing one thing and doing it well (like the [Zettelkasten](https://zettelkasten.de/overview/) and the [Unix](https://en.wikipedia.org/wiki/Unix_philosophy) Philosophies (and just general object oriented thinking).  
I'm happier to have the different .md files for different applications as it makes it easier to write and debug, thus this RFE and my appreciation for @aptend's work (which I'll test tomorrow as I only just updated my plugins as I've been traveling.  Thanks so much!  Looked over the Release notes and this is going to be a game changer :) ).

**Comment by Tejeev** [comment]
@zanodor Thank you, but again, this is not a helpful recommendation for me.  I've mentioned most of this in my previous post so I'll bullet point the reasons for ease of reading at-a-glance

1. I am not at all concerned with the new line character.  I don't need to replace them to paste them in because I just paste as plane text with chars visible.
2. Having all the templates in one file generally bad practice. Please refer to the standard practices I linked in my previous message which are basically as old as computing is and still considered best practices.  They're even the core of Object Oriented Programming (Unix Philosophies) and Obsidian (Zettelkasten).  Following these philosophies make things easier to troubleshoot and work on.
3. You're suggesting I add another plugin which I just don't need.  Each dependency you add to a system is another point of failure and if it's not needed or used only for one or two things, it's better to find another way.  Especially if the dependency is in support of bad practices.  Just use emacs or vim or some other CLI tool to edit the configs if you need to manage hidden chars.  That's not what Obsidian is for and it's easier via CLI once you are in the 
4. It's way more work to do it the way you're proposing in that it requires more keystrokes, mouse interaction, and thought than creating a new template and adding a line to the typing transformer config to call it (all of which can be done by keyboard with minimal work and no need of other files or copying and pasting).

I prefer the RFE filed here which was helpfully addressed by aptend.  
To put it simply, putting all your templates in one file is just bad practice, no matter how or where you configure them and this RFE (Request For Enhancement)/FR addresses that need.  Not anything to do with `/n`.   I do not want to add superfluous (to my uses) plugins and I want to follow standard best practices.  aptend has made that possible so I do not require additional gui complications (they may not seem like complications to you, but I assure you they are relative to my existing workflows).

Again, thank you for your consideration, but it is unrelated to the reasons for this request.

**Comment by Tejeev** [comment]
@aptend this works perfectly :)  I've not yet tested to see if it will implement Templater code (so like `tp` commands per https://silentvoid13.github.io/Templater/) but if not, that may be an RFE/FR I file down the line.  For now I'm a happy camper, so thank you!
